app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
```

Comment:
convention code Ruby "thích" sự đơn giản, có thể bỏ ngoặc `()` khi gọi method ở những chỗ thích hợp
vd như ở line 11 này thì có thể bỏ đi đc em nhé
```suggestion
    @user = User.new user_params
```